dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1176,6 +1176,20 @@ public List<Contentlet> searchByIdentifier(String luceneQuery, int limit, int of
         return contents;
     }
 
+    @Override
+    public List<Contentlet> getAllContentByVariants(final User user,
+            final boolean respectFrontendRoles, final String...variantNames) throws DotDataException, DotStateException,
+            DotSecurityException {
+
+        final String OR = Arrays.stream(variantNames).map((variant)->"variant:"+variant)
```

Comment:
I don't know if this is a good variable name

Reply:
Done